src/mame/atari/atarigt.cpp

```diff
@@ -764,7 +776,6 @@ static INPUT_PORTS_START( primrage )
 INPUT_PORTS_END
 
 
-
 /*************************************
```

Comment:
Please back out this change – the file uses three blank lines to separate sections.